src/sdk-reference/cpp/1/protocol/add-listener/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: sdk.html.hbs
+title: addListener
+description: Add a listener to an event
+---
+
+# addListener
+
+Adds a listener to an event.
+
+## Signature
```

Comment:
Wrong documentation template.

This should be:

```cpp

## Arguments

`<function signature>`

<arguments table>

[optional: ### <argument>
... further explanations]
```

This must be fixed everywhere.

Reply:
On outdated documentation, it's like this. See our debriefing here: https://kaliop.slack.com/archives/GBJ9VRA01/p1541422858016200

The new format has been applied since then, and a global pass will be performed to standardize articles written before these decisions.

SDK Examples:
https://docs-v2.kuzzle.io/sdk-reference/js/6/collection/create/
https://docs-v2.kuzzle.io/sdk-reference/js/6/ms/georadiusbymember/

API: all documentation pages follow the new format

Reply:
Hum ok, great... 
I will do this in the last commit, after fixing the other things.